Formula/sratoolkit.rb

```diff
@@ -37,93 +28,33 @@ class Sratoolkit < Formula
-      end
-    end
-
-    ncbi_vdb_source = buildpath/"ncbi-vdb-source"
-    ncbi_vdb_build = buildpath/"ncbi-vdb-build"
-    ncbi_vdb_source.install resource("ncbi-vdb")
-    cd ncbi_vdb_source do
-      # Fix detection of hdf5 library on macOS as Apple Clang linker doesn't
-      # allow mixing static (-Wl,-Bstatic) and dynamic (-Wl,-Bdynamic) libraries
-      inreplace "setup/konfigure.perl", "-Wl,-Bstatic -lhdf5 -Wl,-Bdynamic", "-lhdf5" if OS.mac?
-
-      # Fix Linux error: `pshufb' is not supported on `generic64.aes'
-      # Upstream ref: https://github.com/ncbi/ncbi-vdb/issues/14
-      inreplace "libs/krypto/Makefile", "-Wa,-march=generic64+aes", "" if OS.linux?
-
-      system "./configure", "--prefix=#{buildpath}/ncbi-vdb-prefix",
-                            "--build=#{ncbi_vdb_build}",
-                            "--with-ngs-sdk-prefix=#{ngs_sdk_prefix}",
-                            *with_formula_args
-      ENV.deparallelize { system "make" }
-    end
-
-    # Fix the error: ld: library not found for -lmagic-static
-    # Upstream PR: https://github.com/ncbi/sra-tools/pull/105
-    inreplace "tools/copycat/Makefile", "-smagic-static", "-smagic"
-
-    # Fix detection of hdf5 library on macOS as Apple Clang linker doesn't
-    # allow mixing static (-Wl,-Bstatic) and dynamic (-Wl,-Bdynamic) libraries
-    inreplace "setup/konfigure.perl", "-Wl,-Bstatic -lhdf5 -Wl,-Bdynamic", "-lhdf5" if OS.mac?
-
-    # Fix the error: utf8proc.o: linker input file unused because linking not done
-    # Upstream issue: https://github.com/ncbi/sra-tools/issues/283
-    if OS.linux?
-      inreplace "tools/driver-tool/utf8proc/Makefile",
-                "$(CC) $(LDFLAGS) -shared",
-                "#{ENV.cc} $(LDFLAGS) -shared"
-    end
-
-    system "./configure", "--prefix=#{prefix}",
-                          "--build=#{buildpath}/sra-tools-build",
-                          "--with-ngs-sdk-prefix=#{ngs_sdk_prefix}",
-                          "--with-ncbi-vdb-sources=#{ncbi_vdb_source}",
-                          "--with-ncbi-vdb-build=#{ncbi_vdb_build}",
-                          *with_formula_args
-    system "make", "install"
-
-    # Remove non-executable files.
-    rm_r [bin/"magic", bin/"ncbi"]
+    ENV.append_to_cflags "-DH5_USE_110_API"
+
+    (buildpath/"ncbi-vdb-source").install resource("ncbi-vdb")
+    system "cmake", "-S", "ncbi-vdb-source", "-B", "ncbi-vdb-build",
+                    "-DNGS_INCDIR=#{buildpath}/ngs/ngs-sdk",
+                    *std_cmake_args
+    system "cmake", "--build", "ncbi-vdb-build"
+
+    system "cmake", "-S", ".", "-B", "sra-tools-build",
+                    "-DVDB_BINDIR=#{buildpath}/ncbi-vdb-build",
+                    "-DVDB_LIBDIR=#{buildpath}/ncbi-vdb-build/lib",
+                    "-DVDB_INCDIR=#{buildpath}/ncbi-vdb-source/interfaces",
+                    *std_cmake_args
+    system "cmake", "--build", "sra-tools-build"
+    system "cmake", "--install", "sra-tools-build"
```

Comment:
I think the upstream has hardcoded `lib64` as installation path. Need to figure out whether to inreplace or symlink directory.

Ref: https://github.com/ncbi/sra-tools/blob/3.0.0/build/env.cmake#L365